Applications/WebPortal/WebApp/library/source/pages/account_directory_page/local_group_suggestion_model.tsx

```diff
@@ -15,19 +15,17 @@ export class LocalGroupSuggestionModel extends GroupSuggestionModel {
   public isLoaded(): boolean {
     return this._isLoaded;
   }
-  // Loads this model.
+
   public async load(): Promise<void> {
     this._isLoaded = true;
   }
 
-  // Returns all the accounts with the given prefix.
   public async loadSuggestions(
       prefix: string): Promise<Beam.DirectoryEntry[]> {
     if(!this.isLoaded) {
       throw Error('Model not loaded.');
     }
     return new Promise<Beam.DirectoryEntry[]> ((resolve) => {
```

Comment:
Eliminate the use of ```new Promise```. This should work by strictly returning a value without wrapping it in a Promise. Try it out and let me know.